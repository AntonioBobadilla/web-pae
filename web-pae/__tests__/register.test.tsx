import Register from '@/pages/register';
import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
import React from 'react';

describe('Register', () => {
  it('renders register page unchanged', () => {
    const { container } = render(<Register />);
    expect(container).toMatchSnapshot();
  });
});
