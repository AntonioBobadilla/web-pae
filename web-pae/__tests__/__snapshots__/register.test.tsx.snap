// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Register renders register page unchanged 1`] = `
<div>
  <div>
    <main>
      <div
        class="mainContainer"
      >
        <div
          class="leftCont"
        >
          <img
            class="registerImage"
            src="/images/register-image.jpg"
          />
        </div>
        <div
          class="register"
        >
          <div
            class="paeRegister"
          >
            <img
              class="paeLogo"
              src="/images/pae-logo.png"
            />
            <h1
              class="paeText"
            >
               PAE | REGISTRO
            </h1>
          </div>
          <div
            class="registerOptions"
          >
            <div
              class="Button"
            >
              <button
                class="container button primary fontS"
              >
                QUIERO UNA ASESORÍA
              </button>
            </div>
            <div
              class="Button"
            >
              <button
                class="container button secondary fontS"
              >
                QUIERO DAR UNA ASESORÍA
              </button>
            </div>
            <h2
              class="language"
            >
              Idioma / Language
            </h2>
            <div
              class="languageOptions"
            >
              <div
                class="toggle"
              >
                <div
                  class="toggle"
                >
                  <img
                    class="flag"
                    src="/images/mxflag.png"
                  />
                  <p
                    class="description"
                  >
                     
                    Español
                     
                  </p>
                </div>
              </div>
              <div
                class="toggle"
              >
                <div
                  class="toggle"
                >
                  <img
                    class="flag"
                    src="/images/usaflag.png"
                  />
                  <p
                    class="description"
                  >
                     
                    English
                     
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </main>
  </div>
</div>
`;
